python-sdk/nuscenes/map_expansion/map_api.py

```diff
@@ -655,6 +664,27 @@ def __init__(self,
         self.canvas_min_y = 0
         self.canvas_aspect_ratio = (self.canvas_max_x - self.canvas_min_x) / (self.canvas_max_y - self.canvas_min_y)
 
+    def render_connectivity(self, resolution_meters: float, figsize: Tuple[int, int]) -> None:
+        """
+        Render the connectivity of all lanes and lane connectors.
+        :param resolution_meters: How finely to discretize the lane. Smaller values ensure curved
+            lanes are properly represented.
+        :param figsize: Size of the figure.
+        """
+        # Discretize all lanes ann lane connectors.
+        nusc_map = self.map_api
+        pose_lists = []
+        for lane in nusc_map.lane + nusc_map.lane_connector:
+            my_lane = nusc_map.arcline_path_3.get(lane['token'], [])
+            discretized = np.array(discretize_lane(my_lane, resolution_meters))
+            pose_lists.append(discretized)
+
+        # Render connectivity lines.
```

Comment:
Seems like you have two fcns in one (discretize centerlines and then render). Might be worth it to split.

Reply:
Okay, I split the two and renamed it to `render_centerlines`.